bigone/__init__.py,sha256=yRbcoSpYKT2ua0wT4c2MiNxYpZZUYH1Ofun_H9gnECI,93
bigone/client.py,sha256=qDgbFHap9cb43V45AoffD-MCvMTaiw33i8zBtu0odII,21233
bigone/exceptions.py,sha256=2FS2fDnllouoSGkv-VPMXCLLCUQq7T0bT492YzLk1IE,1247
python_bigone-0.0.1.dist-info/DESCRIPTION.rst,sha256=OCTuuN6LcWulhHS3d5rfjdsQtW22n7HENFRh6jC6ego,10
python_bigone-0.0.1.dist-info/METADATA,sha256=1AvL-1yBNAxcIveMHuaqk_2bcdiWdL-FF-CRl5PCCRc,969
python_bigone-0.0.1.dist-info/RECORD,,
python_bigone-0.0.1.dist-info/WHEEL,sha256=kdsN-5OJAZIiHN-iO4Rhl82KyS0bDWf4uBwMbkNafr8,110
python_bigone-0.0.1.dist-info/metadata.json,sha256=c1J-FPZPzMfblkNNfmq9dpkyDbLaSqzJIH91hCpi66Y,1106
python_bigone-0.0.1.dist-info/top_level.txt,sha256=oIrxLUvBbRLG80guL5XQ-9U6E1Y58Yo8ywT1hnStwys,7
